code/game/machinery/recharger.dm

```diff
@@ -124,6 +124,13 @@
 					use_power(200 * recharge_coeff)
 					using_power = 1
 
+		if(istype(charging, /obj/item/weapon/tank/internals/rebreather))
+			var/obj/item/weapon/tank/internals/rebreather/B = charging
+			if(B.tcell)
+				if(B.tcell.give(B.tcell.chargerate * recharge_coeff))
+					use_power(200 * recharge_coeff)
+					using_power = 1

```

Comment:
It's nice seeing a new battery powered device. But there's gonna be merge conflicts with the inducer.  I just got done refactoring it to use get_cell(), though, so this whole block won't be necessary if rebreathers are in allowed_devices.